grammar.js

```diff
@@ -82,6 +83,11 @@ module.exports = grammar({
     $._lhs,
   ],
 
+  conflicts: $ => [
+    [$._lhs, $.call],
+    [$._lhs, $.call, $.command_call, $.command_call_with_block],
+  ],
```

Comment:
I'm a little worried about how this conflict will behave, since it seems like it may come up in more cases than just an identifier followed by a `[`. It may be necessary, but I'd like to look into this some, to see if it can be avoided.